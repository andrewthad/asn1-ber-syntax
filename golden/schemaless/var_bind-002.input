30090604 2B819D67 41017F
