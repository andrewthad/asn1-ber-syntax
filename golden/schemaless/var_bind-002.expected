Value
    { tagClass = Universal
    , tagNumber = 16
    , contents = Constructed
        ( fromListN 2
            [ Value
                { tagClass = Universal
                , tagNumber = 6
                , contents = Primitive
                    [ 0x2b
                    , 0x81
                    , 0x9d
                    , 0x67
                    ]
                }
            , Value
                { tagClass = Application
                , tagNumber = 1
                , contents = Primitive [ 0x7f ]
                }
            ]
        )
    }
